# The state of the GPIO board is the state of all the pins
uint8 PIN_COUNT = 40
GPIOPin[] pins
